### PR TITLE
Adding missing roles for eo sa fixing failing e2e tests

### PR DESCRIPTION
**Pull request by ewolinetz**
Follow up to https://github.com/openshift/elasticsearch-operator/pull/76

### CODE REVIEW

File: hack/test-e2e.sh [hunk 1]
```diff
@@ -20,6 +20,8 @@ popd
 # update the manifest with the image built by ci
 sed -i "s,quay.io/openshift/origin-elasticsearch-operator:latest,${IMAGE_ELASTICSEARCH_OPERATOR}," ${manifest}
 
+sudo sysctl -w vm.max_map_count=262144 ||:
```
Comment (richm) [inline hack/test-e2e.sh]: need to skip this if `REMOTE_CLUSTER` is true

Comment (ewolinetz) [reply hack/test-e2e.sh]: updated to skip as necessary

**Pull request merged**